backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BuiltInFictitiousFunctionIrClassFactory.kt

```diff
@@ -126,15 +126,15 @@ internal class BuiltInFictitiousFunctionIrClassFactory(
             builtClassesMap.getOrPut(descriptor) {
                 createClass(descriptor).apply {
                     val functionClass = this
-                    descriptor.declaredTypeParameters.mapTo(typeParameters) { typeParameterDescriptor ->
+                    typeParameters += descriptor.declaredTypeParameters.map { typeParameterDescriptor ->
```

Comment:
What's this for?

Reply:
Ah, I get it, it's because `typeParameters` is not just `List`, not `MutableList`.